auraed/src/runtime/error.rs

```diff
@@ -2,27 +2,40 @@ use log::error;
 use thiserror::Error;
 use tonic::Status;
 
-#[derive(Error, Debug, PartialEq, Eq)]
-pub enum CellServiceError {
+pub(crate) type Result<T> = std::result::Result<T, RuntimeError>;
+
+#[derive(Error, Debug)]
+pub(crate) enum RuntimeError {
     #[error("missing argument in request: {arg}")]
     MissingArgument { arg: String },
 
+    #[error(transparent)]
+    ValidationError(#[from] validation::ValidationError),
+
     #[error("internal error: {msg}: {err}")]
     Internal { msg: String, err: String },
+
+    #[error(transparent)]
+    CgroupsError(#[from] cgroups_rs::error::Error),
+
+    #[error(transparent)]
+    Other(#[from] anyhow::Error),
```

Comment:
Ideally we will eventually eliminate anyhow (I think)

Reply:
i agree.. though it's convenient for creating these quick errors from a string.  i think if we stick to `anyhow!` _only_ we might be ok.